src/model/HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl.hpp

```diff
@@ -0,0 +1,328 @@
+
+      bool setHumidityRatioEquationCoefficient6(double humidityRatioEquationCoefficient6);
+
+      bool setHumidityRatioEquationCoefficient7(double humidityRatioEquationCoefficient7);
+
+      bool setHumidityRatioEquationCoefficient8(double humidityRatioEquationCoefficient8);
+
+      bool
+        setMinimumRegenerationInletAirHumidityRatioforHumidityRatioEquation(double minimumRegenerationInletAirHumidityRatioforHumidityRatioEquation);
+
+      bool
+        setMaximumRegenerationInletAirHumidityRatioforHumidityRatioEquation(double maximumRegenerationInletAirHumidityRatioforHumidityRatioEquation);
+
+      bool setMinimumRegenerationInletAirTemperatureforHumidityRatioEquation(double minimumRegenerationInletAirTemperatureforHumidityRatioEquation);
+
+      bool setMaximumRegenerationInletAirTemperatureforHumidityRatioEquation(double maximumRegenerationInletAirTemperatureforHumidityRatioEquation);
+
+      bool setMinimumProcessInletAirHumidityRatioforHumidityRatioEquation(double minimumProcessInletAirHumidityRatioforHumidityRatioEquation);
+
+      bool setMaximumProcessInletAirHumidityRatioforHumidityRatioEquation(double maximumProcessInletAirHumidityRatioforHumidityRatioEquation);
+
+      bool setMinimumProcessInletAirTemperatureforHumidityRatioEquation(double minimumProcessInletAirTemperatureforHumidityRatioEquation);
+
+      bool setMaximumProcessInletAirTemperatureforHumidityRatioEquation(double maximumProcessInletAirTemperatureforHumidityRatioEquation);
+
+      bool setMinimumRegenerationAirVelocityforHumidityRatioEquation(double minimumRegenerationAirVelocityforHumidityRatioEquation);
+
+      bool setMaximumRegenerationAirVelocityforHumidityRatioEquation(double maximumRegenerationAirVelocityforHumidityRatioEquation);
+
+      bool setMinimumRegenerationOutletAirHumidityRatioforHumidityRatioEquation(
+        double minimumRegenerationOutletAirHumidityRatioforHumidityRatioEquation);
+
+      bool setMaximumRegenerationOutletAirHumidityRatioforHumidityRatioEquation(
+        double maximumRegenerationOutletAirHumidityRatioforHumidityRatioEquation);
+
+      bool setMinimumRegenerationInletAirRelativeHumidityforHumidityRatioEquation(
+        double minimumRegenerationInletAirRelativeHumidityforHumidityRatioEquation);
+
+      bool setMaximumRegenerationInletAirRelativeHumidityforHumidityRatioEquation(
+        double maximumRegenerationInletAirRelativeHumidityforHumidityRatioEquation);
+
+      bool setMinimumProcessInletAirRelativeHumidityforHumidityRatioEquation(double minimumProcessInletAirRelativeHumidityforHumidityRatioEquation);
+
+      bool setMaximumProcessInletAirRelativeHumidityforHumidityRatioEquation(double maximumProcessInletAirRelativeHumidityforHumidityRatioEquation);
+
+      //@}
+      /** @name Other */
+      //@{
+
+      boost::optional<double> autosizedNominalAirFlowRate();
+
+      boost::optional<double> autosizedNominalAirFaceVelocity();
+
+      void autosize();
+
+      void applySizingValues();
```

Comment:
These are virtual overrides and should be labeled appropriately. Move them to the virtual section atop the file and do


```c++
      virtual void autosize() override;

      virtual void applySizingValues() override;
```

Reply:
@jmarrec Hmm, now I disagree. ResourceObject does not have these defined.

Reply:
Oh it's a ResourceObject, right, I lost track of that. It's somewhat bad news for polymorphism, but it's not a big deal. **Do make sure to add it to Model_Impl::autosize** though...

https://github.com/NREL/OpenStudio/blob/aae01b996a4ec6c5c7ec6e9cd0cd3e2de966023e/src/model/Model.cpp#L969-L998